src/Components/CookieConsent.js

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import * as Scrivito from "scrivito";
+import ReactCookieConsent from "react-cookie-consent";
+import cookieConsentIcon from "../assets/images/cookie_consent_icon.svg";
+
+function CookieConsent() {
+  const root = Scrivito.Obj.root();
+
+  if (!root) {
+    return null;
+  }
+
+  const cookieConsentLink = root.get("cookieConsentLink");
```

Comment:
This is missing:

```js
if (!cookieConsentLink) {
  return null;
}
```

Otherwise the component will fail, if no `cookieConsentLink` attribute is set.